packages/metro-minify-uglify/src/minifier.js

```diff
@@ -23,17 +23,19 @@ import type {BabelSourceMap} from '@babel/core';
 function noSourceMap(
   code: string,
   options?: MinifyOptions = {},
+  uglifyModule: ?Object,
 ): ResultWithoutMap {
-  return minify(code, undefined, options).code;
+  return minify(code, undefined, options, uglifyModule).code;
 }
 
 function withSourceMap(
   code: string,
   sourceMap: ?BabelSourceMap,
   filename: string,
   options?: MinifyOptions = {},
+  uglifyModule: ?Object,
```

Comment:
Types need updating [here](https://github.com/facebook/metro/blob/master/packages/metro-minify-uglify/src/types.js.flow#L31) to add the extra `uglifyModule` arg.